### PR TITLE
Comply with our article grouping convention

### PR DESCRIPTION
**Pull request by commuterjoy**
We have a convention of putting groups of articles in to this HTML hierarchy `section -> article -> a`

### CODE REVIEW

File: views/partials/related/more-ons.html [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+<section>
+	<div
+		role="region"
+		data-trackable="more-on-{{@index}}"
+		aria-labelledby="more-on-{{@index}}-title">
+		
+		<div class="more-on">
+			<div class="pod">
+				<div class="header">
+					<span class="header__title">
```
Comment (i-like-robots) [inline views/partials/related/more-ons.html]: not your error but this is invalid

Comment (keirog) [inline views/partials/related/more-ons.html]: Context: http://w3c.github.io/html/introduction.html#restrictions-on-content-models-and-on-attribute-values

Comment (commuterjoy) [inline views/partials/related/more-ons.html]: shall I swap it for a `div` then?

**Comment by kavanagh** [comment]
I don't understand what this is. Should the PR be close or resurrected at some point?

**Comment by commuterjoy** [comment]
yup, old

**Pull request closed without merging**